docs/data-ai/train/capture-images.md

```diff
@@ -0,0 +1,283 @@
+---
+linkTitle: "Capture images for training"
+title: "Capture images for training"
+weight: 20
+layout: "docs"
+type: "docs"
+description: "Capture images that you can use to train a machine learning model."
+---
+
+## Prerequisites
+
+{{% expand "A machine connected to Viam" %}}
+
+{{% snippet "setup.md" %}}
+
+{{% /expand %}}
+
+{{% expand "A camera, connected to your machine, to capture images" %}}
+
+Follow the guide to configure a [webcam](/operate/reference/components/camera/webcam/) or similar [camera component](/operate/reference/components/camera/).
+
+{{% /expand%}}
+
+{{< alert title="Tip" color="tip" >}}
+
+For the best results, use the same camera for both training data capture and production deployment.
+
+{{< /alert >}}
+
+## Capture individual images
+
+{{< tabs >}}
+{{% tab name="Web UI" %}}
+
+You can add images to a dataset directly from a camera or vision component feed in the machine's **CONTROL** or **CONFIGURATION** tabs.
+
+To add an image directly to a dataset from a visual feed, complete the following steps:
+
+1. Open the **TEST** panel of any camera or vision service component to view a feed of images from the camera.
```

Comment:
Seemed not to work for me if I had it on live--maybe a bug/button should be disabled if live. Are you already on this? Otherwise LMK if I should make an eng ticket
```suggestion
1. Open the **TEST** panel of any camera or vision service component to view a feed of images from the camera.
1. Use the refresh frequency dropdown to select an option _other than_ **Live**.
```

Reply:
Works for me, so if you're seeing an issue, you should file a ticket. (un?)fortunately it all works for me properly.

Reply:
Okay; I messaged team-viz (since this was persistent the other day but maybe they already fixed it since I can't repro)